consent-ui/src/app/ais/password-input-page/password-input-page.component.html

```diff
@@ -1 +1,63 @@
-<p>password-input-page works!</p>
+<div class="d-md-flex align-items-center h-100 align-items-stretch">
+
+  <!-- Logo -->
+  <div class="bg-sm-light sticky-top">
+    <nav class="navbar navbar-light text-primary py-3">
+      <a class="navbar-brand" href="#">
+        <img class="logo" [src]="" />
+      </a>
+    </nav>
+  </div>
+  <!-- Consent info -->
+  <div class="col-md-4 p-0 bg-light">
+    <div class="d-md-flex align-items-center h-100 justify-content-center">
+      <consent-app-account-details></consent-app-account-details>
+    </div>
+  </div>
+
+  <!-- Password input -->
+  <div class="col-md-8 bg-white h-md-100">
+    <div class="d-md-flex bg-white h-md-100">
+      <div class="col">
+        <div class="d-md-flex align-items-center h-100 p-4 justify-content-center">
+          <div class="d-md-flex align-items-center">
+            <form [formGroup]="passwordForm" (ngSubmit)="submit()">
+
+              <div class="card">
+                <div class="card-body">
+                  <h2 class="card-title text-center font-weight-bold">Input PIN</h2>
+                  <div class="font-weight-light my-4 text-muted">Please enter pin to you bank in order to confirm the consent.</div>
+
+
+                  <div class="form-group">
+                    <label for="pin" class="text-muted">Pin:</label><br>
+                    <input type="password"
+                           id="pin"
+                           name="pin"
+                           formControlName="pin"
+                           placeholder="Enter your pin"
+                           [ngClass]="{ 'is-invalid': passwordForm.controls['pin'].touched && passwordForm.controls['pin'].errors }"
+                           class="form-control">
+                    <div class="invalid-feedback"
+                         *ngIf="passwordForm.controls['pin'].touched && passwordForm.controls['pin'].hasError('required')">
```

Comment:
In Dynamic Sandbox line like this occurs many times, same applies to our UI, I think we should have separate validation helper component (Just a note, out of scope for this PR)